### PR TITLE
Pass publisher's AdMob personalization perf via the adapters

### PR DESCRIPTION
**Pull request by chauduyphanvu1**
Usage instruction:

```obj-c
AdMobGlobalMediationSettings *mediationSettings = [[AdMobGlobalMediationSettings alloc] init];
[mediationSettings setNpaPref:@"1"];
```

### CODE REVIEW

File: AdColony/AdColonyAdvancedBidder.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//
```
Comment (rsatrasala) [inline AdColony/AdColonyAdvancedBidder.h]: Looka like AdColony files made to this PR by mistake

File: AdMob/AdMobGlobalMediationSettings.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+#if __has_include(<MoPub/MoPub.h>)
+#import <MoPub/MoPub.h>
+#elif __has_include(<MoPubSDKFramework/MoPub.h>)
+#import <MoPubSDKFramework/MoPub.h>
+#else
+#import "MPMediationSettingsProtocol.h"
+#endif
+
+@interface AdMobGlobalMediationSettings : NSObject
```
Comment (rsatrasala) [inline AdMob/AdMobGlobalMediationSettings.h]: You are not using mediation settings anymore right. Can we clean it up?

File: AdMob/MPGoogleAdMobBannerCustomEvent.m [hunk 3]
```diff
@@ -8,6 +8,8 @@
 #import <GoogleMobileAds/GoogleMobileAds.h>
 #import "MPGoogleAdMobBannerCustomEvent.h"
 #import "MPLogging.h"
+#import "AdMobGlobalMediationSettings.h"
```
Comment (rsatrasala) [inline AdMob/MPGoogleAdMobBannerCustomEvent.m]: import is unnecessary

File: AdColony/AdColonyAdvancedBidder.h (hunk 1 above)
Comment (chauduyphanvu1) [reply AdColony/AdColonyAdvancedBidder.h]: Somehow my PR got bundled with the closed AdColony PR. I'll close this, and you can make a new one once done.

**Pull request closed without merging**